scripts/apollo_coverage.sh

```diff
@@ -8,8 +8,9 @@ source "${TOP_DIR}/scripts/apollo_base.sh"
 BAZEL_OUT="${TOP_DIR}/bazel-out" # $(bazel info output_path)
 COVERAGE_HTML="${TOP_DIR}/.cache/coverage"
 COVERAGE_DAT="${BAZEL_OUT}/_coverage/_coverage_report.dat"
-GENHTML_OPTIONS="--rc genhtml_branch_coverage=1"
-#GENHTML_OPTIONS="--rc genhtml_branch_coverage=1 --highlight --legend"
+
+# Note(storypku): branch coverage seems not work when running bazel coverage
+# GENHTML_OPTIONS="--rc genhtml_branch_coverage=1 --highlight --legend"
```

Comment:
Currently, branch coverage data can't be obtained by running bazel coverage.
See https://stackoverflow.com/questions/57770529/bazel-branch-coverage-no-data-found